features/bootstrap/IsolatedProcessContext.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+use Behat\Behat\Tester\Exception\PendingException;
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use PhpSpec\Exception\Example\SkippingException;
+use Symfony\Component\Process\Process;
+
+/**
+ * Defines application features from the specific context.
+ */
+class IsolatedProcessContext implements Context, SnippetAcceptingContext
+{
+    private $lastOutput;
+
+    /**
+     * @Given I have started describing the :class class
+     */
+    public function iHaveStartedDescribingTheClass($class)
+    {
+        $process = new Process($this->buildPhpSpecCmd() . ' describe '. escapeshellarg($class));
+        $process->run();
+
+        expect($process->getExitCode())->toBe(0);
+    }
+
+    /**
+     * @When I run phpspec and answer :answer when asked if I want to generate the code
+     */
+    public function iRunPhpspecAndAnswerWhenAskedIfIWantToGenerateTheCode($answer)
+    {
+        if (!`which expect`) {
+            throw new SkippingException('Cannot test interactive scripts on this platform');
```

Comment:
PhpStorm hides it from me ;-)

I want to skip when `expect` is not available, but it seems to not be possible I think?